modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1308 @@
+
+As a consequence, when child block elements set to `contenteditable="true"` such as a `<p>` tag were nested within the `contenteditable="false"` element root, the editor would try to split the child block element when the user hit the `enter` command. 
+
+As the paragraph tag is intended to be a block for `text`, it is classified as `semantically incorrect` to split a paragraph block element.
+In this particular instance, child elements in {productname} such as the `<p>` tag would split the block element resulting in a new block (such as a new paragraph) to be created inside the paragraph when a user attempted to edit it inside a `contenteditable="false"` root element. 
+
+Fixes were introduced with the release of {productname} 6.4.1, that:
+
+* Prevents editing host elements like `<p contenteditable="true">paragraph text</p>` in a `contenteditable="false"` root to be split.
+* Prevents editing hosts elements inside a `contenteditable="false"` root to be split.
+
+As a result, when a root element is set to `contenteditable="false"`, any `contenteditable="true"` elements inside that root element such as `<p>` tag will no longer try to split the `contenteditable="true"` element. 
+
+=== Creating a list in a table cell when the caret is in front of an anchor element would not properly include the anchor in the list.
+//#TINY-6853
+
+In previous versions of {productname}, a bug was causing issues with `links` while they were nested within table-cells. As a consequence, it was found that when creating a `num-list` or `bullet-list` to the left (node) of a `link` that was nested within a table cell the `list` would *not* include the created `link` within the `num or bullet-list` as expected. 
+
+To fix this bug, the table cell now checks *If* the caret position is in front of an *anchor element*, `if` (true), then the table cell will include the anchor in the `list`.
+
+In {productname} 6.4.1, anchor elements are included inside `lists` while nested within table cells.
+
+=== Dragging and dropping noneditable elements on table borders would remove the element on drop
+//#TINY-9021
+
+In previous versions of {productname}, dragging and dropping `contenteditable="false"` elements on table borders would create issues during the drop action.
+
+As a consequence, this would result in the `contenteditable="false"` element being deleted after the `action` when dropped onto the borderline within the table cell.
+To fix this issue, {productname} 6.4.1, introduced a new `isActive` api to the handler for dragger, that checks if the event `isActive` or not, which then hides the resize bar during the dragging event, which now prevents content from being dropped onto table bars.
+
+In {productname} 6.4.1, dragging an inline `contenteditable="false"` element onto the border inside a table, now moves it into a neighbouring cell instead of deleting it.
```

Comment:
```suggestion
In previous versions of {productname}, dragging a `contenteditable="false"` element and dropping it on a table border would cause the `contenteditable="false"` element to be deleted. The user often intends to drop the element into a table cell, but may accidentally drop it on a table border, undesirably causing the element's deletion.

{productname} 6.4.1 introduced a new `isActive` api for Dragster that checks whether an element is being dragged or not. If an element is being dragged, the resize bars on table borders are hidden, which prevents the deletion side-effect of dropping elements onto the table border resize bars. If an element is dropped onto a table border, it will now be inserted into the left or right adjacent cell instead of being deleted.
```